fish_prompt.fish

```diff
@@ -12,7 +12,7 @@ set -g __pure_fresh_session 1
 
 __pure_set_default pure_symbol_prompt "❯"
 __pure_set_default pure_symbol_git_down_arrow "⇣"
-__pure_set_default pure_symbol_git_down_arrow "⇡"
+__pure_set_default pure_symbol_git_up_arrow "⇡"
```

Comment:
I reckon this comment is on the wrong line and you refer to all the edits that introduce the `string` internal command.